File: dongyimai-manager-web/src/main/webapp/js/controller/indexController.js
 //控制层 
app.controller('indexController' ,function($scope,$controller ,indexService){
	
	$controller('baseController',{$scope:$scope});//继承
	
    //读取列表数据绑定到表单中  
	$scope.showName=function(){
		indexService.showName().success(
			function(response){
				$scope.username=response.username;
			}			
		);
	}    

});	

File: dongyimai-manager-web/src/main/java/com/xxx/proj/controller/IndexController.java
package com.xxx.proj.controller;

import org.springframework.security.core.context.SecurityContextHolder;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;

import java.util.HashMap;
import java.util.Map;

@RestController
@RequestMapping("/index")
public class IndexController {

    @RequestMapping("/showName")
    public Map<String,String> showName() {
        //从安全框架中读取当前登录用户名
        String username = SecurityContextHolder.getContext().getAuthentication().getName();
        Map map = new HashMap();
        map.put("username", username);
        System.out.println(map);
        return map;
    }
}
